db/seeds.rb

```diff
@@ -1,9 +1,98 @@
-# This file should ensure the existence of records required to run the application in every environment (production,
-# development, test). The code here should be idempotent so that it can be executed at any point in every environment.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Example:
-#
-#   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
-#     MovieGenre.find_or_create_by!(name: genre_name)
-#   end
+# Do we want to delete existing data each time
```

Comment:
Good question! Seeing as we will also work with user inputs I'd leave this commented out for now. We can always delete manually in rails console if need be.